src/main/java/me/zeroeightsix/kami/module/modules/render/NoHurtCam.kt

```diff
@@ -2,25 +2,14 @@ package me.zeroeightsix.kami.module.modules.render
 
 import me.zeroeightsix.kami.module.Module
 
-/**
- * @author 086
- */
 @Module.Info(
         name = "NoHurtCam",
         category = Module.Category.RENDER,
         description = "Disables the 'hurt' camera effect"
 )
-class NoHurtCam : Module() {
-    companion object {
-        private var INSTANCE: NoHurtCam? = null
-
-        @JvmStatic
-        fun shouldDisable(): Boolean {
-            return INSTANCE != null && INSTANCE!!.isEnabled
-        }
-    }
-
-    init {
-        INSTANCE = this
+object NoHurtCam : Module() {
+    @JvmStatic
+    fun shouldDisable(): Boolean {
```

Comment:
why is this even a thing? there is no consequences from trying to disable() when something isEnabled